### PR TITLE
improved filters and templates

### PR DESCRIPTION
**Pull request by KowalskiIT**
changelog:

- upgraded filtersets
- changed filter position in templates
- added query for source in income creation/filter view

### CODE REVIEW

File: budget_manager/templates/income_list.html [hunk 1]
```diff
@@ -1,64 +1,55 @@
...
-                    <th class="text-center" scope="col">Amount</th>
-                    <th class="text-center" scope="col">Source</th>
-                    <th class="text-center" scope="col">Date</th>
-                    <th class="text-center" scope="col">Currency</th>
+                    <th class="text-center" scope="col">Nr.</th>
+                    <th class="text-center" scope="col">Wartość</th>
+                    <th class="text-center" scope="col">Źródło</th>
+                    <th class="text-center" scope="col">Data</th>
+                    <th class="text-center" scope="col">Waluta</th>
                     <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/income_list.html:28]: unnecessary tag

File: budget_manager/templates/income_list.html [hunk 2]
```diff
@@ -1,64 +1,55 @@
...
                     <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/income_list.html:29]: unnecessary tag

File: budget_manager/templates/income_list.html [hunk 3]
```diff
@@ -1,64 +1,55 @@
...
+        <form action="" method="get">
+        <table class="table table-striped mt-5">
+            <tr>
+                <th class="text-center col-sm-1" scope="col">Wartość: {{ filter.form.amount }} </th>
+                <th class="text-center col-sm-1" scope="col">Źródło: {{ filter.form.source }}</th>
+                <th class="text-center col-sm-1" scope="col">Od: {{ filter.form.income_date__gte }}
+                                                            Do: {{ filter.form.income_date__lte }}</th>
+                <th class="text-center col-sm-1" scope="col">Waluta: {{ filter.form.currency }}</th>
+                <th><input type="submit" class="d-block btn btn-primary  my-3" value="Szukaj"></th>
+                <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/income_list.html:16]: unnecessary tag

File: budget_manager/templates/expense_filter_list.html [hunk 4]
```diff
@@ -2,16 +2,29 @@
...
-                    <th class="text-center" scope="col">Date</th>
-                    <th class="text-center" scope="col">Currency</th>
-                    <th class="text-center" scope="col">Category</th>
+                    <th class="text-center" scope="col">Nr.</th>
+                    <th class="text-center" scope="col">Nazwa</th>
+                    <th class="text-center" scope="col">Koszt</th>
+                    <th class="text-center" scope="col">Data</th>
+                    <th class="text-center" scope="col">Waluta</th>
+                    <th class="text-center" scope="col">Kategoria</th>
                     <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/expense_filter_list.html:28]: unnecessary tag

File: budget_manager/templates/expense_filter_list.html [hunk 5]
```diff
@@ -2,16 +2,29 @@
...
                     <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/expense_filter_list.html:29]: unnecessary tag

File: budget_manager/templates/expense_filter_list.html [hunk 6]
```diff
@@ -2,16 +2,29 @@
...
+        <form action="" method="get">
+        <table class="table table-striped mt-5">
+            <tr>
+                <th class="text-center col-sm-2" scope="col">Nazwa: {{ filter.form.name}}</th>
+                <th class="text-center col-sm-1" scope="col">Wartość: {{ filter.form.cost}} </th>
+                <th class="text-center col-sm-1" scope="col">Od: {{ filter.form.expense_date__gte}} Do: {{ filter.form.expense_date__lte}}</th>
+                <th class="text-center col-sm-1" scope="col">Waluta: {{ filter.form.currency}}</th>
+                <th class="text-center col-sm-1" scope="col">Kategoria: {{ filter.form.category}}</th>
+                <th><input type="submit" class="d-block btn btn-primary  my-3" value="Szukaj"></th>
+                <th></th>
```
Comment (M4ti06) [inline budget_manager/templates/expense_filter_list.html:15]: unnecessary tag

**Review by M4ti06 — CHANGES_REQUESTED** [review]
Check if you agree with me

**Review by M4ti06 — APPROVED** [review]
good to go

**Comment by KowalskiIT** [comment]
highlited issues will be fixed in upcomming pull request

**Pull request merged**